test/test_torch.py

```diff
@@ -4891,6 +4891,136 @@ def test_diff(self, device, dtype):
             scalar = torch.tensor(2, device=device, dtype=dtype)
             torch.diff(scalar)
 
+    def _test_histogram_numpy(self, t, bins, bin_range, weights, density):
+        def to_np(t):
+            if not torch.is_tensor(t):
+                return t
+            if t.dtype == torch.bfloat16:
```

Comment:
This is now moot because the function (and test) only work with float32 and float64, right?